store/models/log_events.go

```diff
@@ -26,7 +26,8 @@ var (
 	// RunLogTopic is the signature for the RunRequest(...) event
 	// which Chainlink RunLog initiators watch for.
 	// See https://github.com/smartcontractkit/chainlink/blob/master/solidity/contracts/Oracle.sol
-	RunLogTopic = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,bytes)")
+	RunLogTopic  = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,bytes)")
+	RunLogTopic2 = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,address,bytes4,uint256,bytes)")
```

Comment:
This is going to look weird when I add a `RunLogTopic3` to finish this chore: https://www.pivotaltracker.com/story/show/162923035

Reply:
I don't see what's weird here, but the name was supposed to be a placeholder. Putting a date on the names so that there is at least a bit of context as to when it was changed.